backend/README.md

```diff
@@ -23,16 +23,35 @@ To run:
 npm run start:dev
 ```
 
-## Setup .env
+## Setup application.yaml
```

Comment:
Why you switched to an yaml config file?

Reply:
To make application customisation more structured.